src/shared/utils/__tests__/traverseAllChildren-test.js

```diff
@@ -79,6 +80,8 @@ describe('traverseAllChildren', function() {
       '.0'
     );
     expect(traverseContext.length).toEqual(1);
+    expect(console.error.calls.length).toBe(1);
+    expect(console.error.calls[0].args[0]).toContain('Warning: Each child in an array or iterator should have a unique "key" prop.');
```

Comment:
Never mind, I see that the key here is significant in the `.0`. This is fine as-is then.